### PR TITLE
Testbranch

### PR DESCRIPTION
**Pull request by utkarsh-singh-shipsy**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# <img src="distribution/icon.png" width="45" align="left"> Refined GitHub
+# <img src="distribution/icon.png" width="45" align="left"> Standardised PR Process
```
Comment (kunal-kumar-shipsy) [inline readme.md:1]: naming convention
use promise.map 
variable type
------------------------------------------------
ISSUE CODE:
JS001 - Variable naming convention should be correct
JS002 - Define type for each variable
JS003 - Use promise.map here

Comment (rahul-gupta-shipsy) [reply readme.md:1]: rahulgupta

Comment (kunal-kumar-shipsy) [inline readme.md:1]: some issue in code.

ISSUE_CODE:
JS001
JS002

File: package-lock.json [hunk 2]
```diff
@@ -1,11592 +1,6 @@
...
-				"delay": "^5.0.0",
-				"delegate-it": "^3.0.0",
-				"dom-chef": "^5.0.0",
-				"dom-loaded": "^3.0.0",
-				"doma": "^3.0.1",
-				"element-ready": "^6.2.0",
-				"fit-textarea": "^2.0.0",
-				"flat-zip": "^1.0.1",
-				"github-url-detection": "^5.8.0",
-				"image-promise": "^7.0.1",
```
Comment (rahul-gupta-shipsy) [inline package-lock.json:21]: hello variable

Issue Code: 
JS001 
JS002

File: readme.md (hunk 1 above)
Comment (kunal-kumar-shipsy) [inline readme.md:1]: naming convention
use promise.map 
variable type

**Pull request closed without merging**